src/index.js

```diff
@@ -11,6 +13,16 @@ return shiki
         theme: config.theme,
     })
     .then((hl) => {
+        config.languages.forEach(async lang => {
+            const grammarDef = JSON.parse(fs.readFileSync("./" + lang.grammar));
+            const langDef = {
+                id: lang.id,
+                scopeName: lang.scope_name,
+                grammar: grammarDef,
+                aliases: lang.aliases || [],
+            };
+            await hl.loadLanguage(langDef);
+        });
```

Comment:
```suggestion
    .then(async (hl) => {
        for (const lang of config.languages) {
            const grammarDef = JSON.parse(await fsPromises.readFile(lang.grammar));
            const langDef = {
                id: lang.id,
                scopeName: lang.scope_name,
                grammar: grammarDef,
                aliases: lang.aliases || [],
            };
            await hl.loadLanguage(langDef);
        }
```

Using `async/await` in `forEach` loop is not a good idea.

Also, use `fsPromises.readFile` instead of `fs.readFileSync`.